source/hardware/controllers/denon_mc7000.rst

```diff
@@ -21,51 +10,35 @@ analogue (vinyl) and digital sources.
 
 .. versionadded:: 2.2.4
```

Comment:
Could you please describe the functional mapping changes in a few words?

```suggestion
.. versionadded:: 2.2.4
.. versionchanged:: 2.3.0
   Please describe the changes in a few words here.
```

Reply:
i have now updated the changelog. I have used :hwlabel: initially but that looked like a mess. I'd prefer to keep it simple without :hwlabel: inside the changelog. Thanks a lot!